### ISSUE TITLE
[feature request] fullfill the "scope" field of network

### ISSUE THREAD

**Issue by mathspanda** [alibaba/pouch#2501]
## Why you need it?

The value of network scope is empty.
```
root@lin-dev-vm:~# pouch network ls
NETWORK ID   NAME       DRIVER   SCOPE
53abe89df2   host       host            
6da3ff62a7   none       null             
dbc64f8ea8   bridge     bridge    
```  

## How it could be?

1、fullfill scope value returned in pouch ls/inpsect
2、support scope flag when creating network

## Other related information

None

**Comment by allencloud** [comment]
Any update on this? @mathspanda

**Comment by allencloud** [comment]
I think we could close this, since it has been fixed in #2532 . Thanks for your work. @mathspanda
